GestureHandler.js

```diff
@@ -518,6 +522,7 @@ class RectButton extends React.Component {
   static defaultProps = {
     activeOpacity: 0.105,
     underlayColor: 'black',
+    overflow: 'hidden',
```

Comment:
this may not be necessary as we render BaseButton here that already has it set as default